src/include/hardware.h

```diff
@@ -28,29 +28,32 @@
 /*
  * Atomic versions of the flag set/clear macros.
  */
+#define FLD_ISSET_ATOMIC_16(field, mask) (field & (uint16_t)(mask))
 
-#define F_ISSET_ATOMIC_16(p, mask) ((p)->flags_atomic & (uint16_t)(mask))
-
-#define F_SET_ATOMIC_16(p, mask)                                                             \
-    do {                                                                                     \
-        uint16_t __orig;                                                                     \
-        if (F_ISSET_ATOMIC_16(p, mask))                                                      \
-            break;                                                                           \
-        do {                                                                                 \
-            __orig = (p)->flags_atomic;                                                      \
-        } while (!__wt_atomic_cas16(&(p)->flags_atomic, __orig, __orig | (uint16_t)(mask))); \
+#define FLD_SET_ATOMIC_16(field, mask)                                           \
```

Comment:
Could you add brackets around the `field` and `mask` parameters wherever they are used in the following macros